app/src/main/java/com/toolslab/reviewme/ScreenUtils.kt

```diff
@@ -0,0 +1,34 @@
+package com.resocoder.forecastmvvm.utils
+
+import android.content.Context
+import android.util.DisplayMetrics
+import android.view.WindowManager
+
+
+/**
+ * Author errysuprayogi on 09,January,2020
+ */
+object ScreenUtils {
+    fun getScreenHeight(context: Context): Int {
+        val windowManager = context.getSystemService(Context.WINDOW_SERVICE) as WindowManager
+        val dm = DisplayMetrics()
+        windowManager.defaultDisplay.getMetrics(dm)
+        return dm.heightPixels
+    }
+
+    fun getScreenWidth(context: Context): Int {
+        val windowManager = context.getSystemService(Context.WINDOW_SERVICE) as WindowManager
+        val dm = DisplayMetrics()
+        windowManager.defaultDisplay.getMetrics(dm)
+        return dm.widthPixels
+    }
+
+    fun getStatusBarHeight(context: Context): Int {
+        var result = 0
+        val resourceId: Int = context.getResources().getIdentifier("status_bar_height", "dimen", "android")
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/errysuprayogi/ReviewMe/blob/5e6480c9bd0b2e46084f2c3d0e8334ee8ea8a889/app/src/main/java/com/toolslab/reviewme/ScreenUtils.kt#L28"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>